Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py

```diff
@@ -400,6 +419,8 @@ def create_pull_request(cls, repository, args, branch_point, callback=None, unbl
                 return 1
             print("Updated '{}'!".format(pr))
         else:
+            # TODO: I NEED TO BE REMOVED LATER
+            return 1
```

Comment:
This can be removed now, yes?

Reply:
I just caught these from the style checker, they should be gone now